exercises/hamming/hamming_test.exs

```diff
@@ -8,34 +8,46 @@ ExUnit.configure exclude: :pending, trace: true
 defmodule DNATest do
   use ExUnit.Case
 
-  # @tag :pending
+#   @tag :pending
+  test "hamming distance for strands of same length returns :ok" do
+    {code, _} = DNA.hamming_distance('', '')
+    assert code == :ok
```

Comment:
This would probably be more idiomatic as `assert {:ok, _} = DNA.hamming_distance('', '')`.

I'd describe the test slightly different, as it doesn't just return `:ok`. I'd say "hamming distance for strands of the same length are ok"